lib/sentry.dart

```diff
@@ -160,8 +160,23 @@ class SentryClient {
   User userContext;
 
   @visibleForTesting
-  String get postUri =>
-      '${dsnUri.scheme}://${dsnUri.host}/api/$projectId/store/';
+  String get postUri {
+    var port = dsnUri.hasPort &&
+            ((dsnUri.scheme == 'http' && dsnUri.port != 80) ||
+                (dsnUri.scheme == 'https' && dsnUri.port != 443))
+        ? ':${dsnUri.port}'
+        : '';
+    var pathLength = dsnUri.pathSegments.length;
+    var apiPath;
```

Comment:
To make sure the language doesn't infer `dynamic` by mistake, let's put explicit types on the left-hand side (i.e. `String`, `int`, `String`).

Reply:
Updated, thx.